scripts/signing/add_output_to_repo.py

```diff
@@ -0,0 +1,167 @@
+        deb_url = f"{ARTIFACTORY_DEB_REPO_URL}/pool/{base}"
+        dest_opts = f"deb.distribution={args.stage};deb.component=main;deb.architecture=amd64;deb.architecture=arm64"
+        dest_url = f"{deb_url};{dest_opts}"
+        if not args.yes and artifactory_file_exists(deb_url, args.artifactory_user, args.artifactory_token):
+            overwrite = input(f"package {deb_url} already exists. Overwrite? [y/N] ")
+            if overwrite.lower() not in ("y", "yes"):
+                sys.exit(0)
+        orig_metadata_md5 = get_md5_from_artifactory(metadata_api_url, args.artifactory_user, args.artifactory_token)
+        upload_file_to_artifactory(path, dest_url, args.artifactory_user, args.artifactory_token)
+        wait_for_artifactory_metadata(
+            metadata_api_url, orig_metadata_md5, args.artifactory_user, args.artifactory_token, args.timeout
+        )
+
+    return metadata_url
+
+
+def add_rpms_to_repo(paths, args):
+    metadata_api_url = f"{ARTIFACTORY_API_URL}/storage/{ARTIFACTORY_RPM_REPO}/{args.stage}/repodata/repomd.xml"
+    metadata_url = f"{ARTIFACTORY_RPM_REPO_URL}/{args.stage}/repodata/repomd.xml"
+
+    for path in paths:
+        base = os.path.basename(path)
+        dest_url = f"{ARTIFACTORY_RPM_REPO_URL}/{args.stage}/{base}"
+        if not args.yes and artifactory_file_exists(dest_url, args.artifactory_user, args.artifactory_token):
+            overwrite = input(f"package {dest_url} already exists. Overwrite? [y/N] ")
+            if overwrite.lower() not in ("y", "yes"):
+                sys.exit(0)
+        orig_metadata_md5 = get_md5_from_artifactory(metadata_api_url, args.artifactory_user, args.artifactory_token)
+        if args.chaperone_token and args.staging_user and args.staging_token:
+            with tempfile.TemporaryDirectory() as tmpdir:
+                signed_rpm_path = os.path.join(tmpdir, base)
+                print(f"Signing {path} (may take 10+ minutes):")
+                sign_file(
+                    path,
+                    signed_rpm_path,
+                    "RPM",
+                    args.chaperone_token,
+                    args.staging_user,
+                    args.staging_token,
+                    args.timeout,
+                )
+                upload_file_to_artifactory(signed_rpm_path, dest_url, args.artifactory_user, args.artifactory_token)
+        else:
+            upload_file_to_artifactory(path, dest_url, args.artifactory_user, args.artifactory_token)
+        wait_for_artifactory_metadata(
+            metadata_api_url, orig_metadata_md5, args.artifactory_user, args.artifactory_token, args.timeout
+        )
+
+    return metadata_url
+
+
+def main():
+    args = getargs()
+
+    if os.path.isdir(args.path):
+        paths = glob.glob(f"{args.path}/**/*.{args.package_type}", recursive=True)
+    else:
+        paths = [args.path]
+
+    if args.package_type == "deb":
+        metadata_url = add_debs_to_repo(paths, args)
+    else:
```

Comment:
```
elif args.package_type == "rpm":
  ...
else:
   ERROR
```

Reply:
This will be handled by argparse since it will only allow deb or rpm as a choice.